WalletWasabi.Gui/Tabs/WalletManager/LoadWallets/LoadWalletViewModel.cs

```diff
@@ -140,7 +140,7 @@ public override void OnCategorySelected()
 			Password = "";
 		}
 
-		public KeyManager LoadKeyManager()
+		public KeyManager? LoadKeyManager()
```

Comment:
It may be a good idea to rename this method to `TryLoadKeyManager` to signal to the reader that it may not return the result as now may reader expect.